packages/editor/src/pages/Editor/store/settings/utilities.ts

```diff
@@ -20,19 +22,25 @@ export const defaultSettings: ISettings = {
   'editor.wordWrap': 'bounded',
 };
 
+export const invisibleDefaultSettings: { [key: string]: any } = {};
```

Comment:
This seems a bit messy. The intention is to have settings that if people know they exist, it allows them to set it but doesn't show up in the default settings, right? Could we instead check for some key in localStorage that gets set to indicate they are advanced/dev mode, and make the default settings include those special/invisible keys when they are in that mode? Perhaps there could be a button in the footer that allows you to enter this mode? Or the mode itself could be another setting?

Reply:
It's a *bit* messy, but it gets the job done, and is quite practical.  For the settings that I anticipate putting into here, it would be for scenarios where the developer received a set of instructions that, among other things, say "paste these two lines" into the user settings.  Having those settings visible wouldn't help the user one way or another.
So I'm inclined to keep it as is.